### PR TITLE
news post for new global team members

### PR DESCRIPTION
**Pull request by shannonpileggi**

### CODE REVIEW

File: content/news/2024-02-07-welcome-global-team/index.en.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+| ![](img/alyssa_columbus.png)        | Alyssa Columbus           | Chapter Onboarding |
+| ![](img/luana_atunes.png)           | Luana Atunes              | Abstract Review    |
+| ![](img/sayantika_banik.png)        | Sayantika Banik           | Abstract Review    |
+| ![](img/cosima_meyer.png)           | Cosima Meyer              | Website            |
+| ![](img/andrea_gomez_vargas.png)    | Andrea Gómez Vargas       | Website            |
+| ![](img/leena_el_seed.jpg)          | Leena El Seed             | Conference Liaison |
+| ![](img/daniela_vazquez.png)        | Daniela Vázquez           | Social Media       |
+
+# Onboarding process
+
```
Comment (drmowinckels) [inline content/news/2024-02-07-welcome-global-team/index.en.md:46]:
```suggestion
The global governance of R-Ladies is seeing increased activity as we all  slowly are
coming out of 4 difficult years. The new members of the Global team are vital to
help support and grow our community.

```

**Review by drmowinckels — COMMENTED** [review]
Added some text after all the members.

**Review by drmowinckels — APPROVED** [review]

**Comment by shannonpileggi** [comment]
Note: we might have two more members to add, not ready for merge

File: content/news/2024-02-07-welcome-global-team/index.en.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+# What the future holds
+
+We are excited to see what the future holds for R-Ladies with these new
+members of the Global Team. Having new people, with fresh ideas and 
+perspectives, will help us to continue to grow and improve as an organization.
+We are looking forward to working with these new members, and we hope
+that you will join us in welcoming them to the Global Team!
+
+You can see an overview of the entire global team [here](/about-us/global-team/).
```
Comment (shannonpileggi) [inline content/news/2024-02-07-welcome-global-team/index.en.md:65]:
```suggestion
You can see an overview of the entire global team at [About -> Meet the Global Team](/about-us/global-team/).
```

File: content/news/2024-02-07-welcome-global-team/index.en.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+| ![](img/luana_atunes.png)           | Luana Atunes              | Abstract Review    |
+| ![](img/sayantika_banik.png)        | Sayantika Banik           | Abstract Review    |
+| ![](img/cosima_meyer.png)           | Cosima Meyer              | Website            |
+| ![](img/andrea_gomez_vargas.png)    | Andrea Gómez Vargas       | Website            |
+| ![](img/leena_el_seed.jpg)          | Leena El Seed             | Conference Liaison |
+| ![](img/daniela_vazquez.png)        | Daniela Vázquez           | Social Media       |
+
+# Onboarding process
+The global governance of R-Ladies is seeing increased activity as we all  slowly are
+coming out of 4 difficult years. The new members of the Global team are vital to
```
Comment (shannonpileggi) [inline content/news/2024-02-07-welcome-global-team/index.en.md:47]:
```suggestion
coming out of four difficult years. The new members of the Global team are vital to
```

File: content/news/2024-02-07-welcome-global-team/index.en.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+|                                     |                           |                    |
+|-------------------------------------|---------------------------|--------------------|
+| ![](img/glenda_mendieta.jpg)        | Glenda Mendieta           | Translation        |
+| ![](img/sara_acevedo.jpg)           | Nicola Rennie             | Campaigns          |
+| ![](img/sara_acevedo.jpg)           | Sara Acevedo              | Code of Conduct    |
+| ![](img/hebah_bukhari.jpg)          | Hebah Bukhari             | Community Slack    |
+| ![](img/priyanka_gagneja.jpg)       | Priyanka Gagneja          | Community Slack    |
+| ![](img/renata_hirota.png)          | Renata Hirota             | Blog               |
+| ![](img/virginia_garcia_alonso.jpg) | Virginia A. García Alonso | Mentoring          |
+| ![](img/nicola_rennie.jpg)          | Nic Crane                 | Meetup Pro         |
```
Comment (drmowinckels) [inline content/news/2024-02-07-welcome-global-team/index.en.md:43]:
```suggestion
| ![](img/nic_crane.jpg)          | Nic Crane                 | Meetup Pro         |
```

File: content/news/2024-02-07-welcome-global-team/index.en.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+support us with their time and effort to champion our [mission](https://rladies.org/about-us/mission/).
+
+
+
+# New members
+
+|                                     |                           |                    |
+|-------------------------------------|---------------------------|--------------------|
+| ![](img/glenda_mendieta.jpg)        | Glenda Mendieta           | Translation        |
+| ![](img/sara_acevedo.jpg)           | Nicola Rennie             | Campaigns          |
```
Comment (drmowinckels) [inline content/news/2024-02-07-welcome-global-team/index.en.md:37]:
```suggestion
| ![](img/nicola_rennie.jpg)           | Nicola Rennie             | Campaigns          |
```

**Review by drmowinckels — CHANGES_REQUESTED** [review]
Fic Nicola and Nic's images

File: content/news/2024-02-07-welcome-global-team/index.en.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+| ![](img/sayantika_banik.png)        | Sayantika Banik           | Abstract Review    |
+| ![](img/cosima_meyer.png)           | Cosima Meyer              | Website            |
+| ![](img/andrea_gomez_vargas.png)    | Andrea Gómez Vargas       | Website            |
+| ![](img/leena_el_seed.jpg)          | Leena El Seed             | Conference Liaison |
+| ![](img/daniela_vazquez.png)        | Daniela Vázquez           | Social Media       |
+
+# Onboarding needs
+
+Between 2020-2023, membership R-Ladies global organization increased 
+while the availability of volunteers remained limited. At the conclusion of this
```
Comment (shannonpileggi) [inline content/news/2024-02-07-welcome-global-team/index.en.md:55]:
```suggestion
Between 2020-2023, membership in the R-Ladies global organization increased 
while the availability of volunteers remained limited. At the conclusion of this
```

File: content/news/2024-02-07-welcome-global-team/index.en.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+author: "Leadership Team"
+type: blog
+date: "2024-02-07"
+output: html_document
+image: img/welcome.PNG
+tags: 
+- community
+categories: 
+- R-Ladies
+summary: R-Ladies Global Team has 13 new members!
```
Comment (rivaquiroga) [inline content/news/2024-02-07-welcome-global-team/index.en.md:12]:
```suggestion
summary: R-Ladies Global Team has 15 new members!
```

File: content/news/2024-02-07-welcome-global-team/index.en.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+slug: /news/new-to-global-team-2023/
+---
+
+![Thumbnail photos of 13 new global team members on purple background with text `Welcome`.](img/welcome.PNG)
```
Comment (rivaquiroga) [inline content/news/2024-02-07-welcome-global-team/index.en.md:16]:
```suggestion
![Thumbnail photos of 15 new global team members on purple background with text `Welcome`.](img/welcome.PNG)
```

File: content/news/2024-03-13-welcome-global-team/index.en.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+# Onboarding needs
+
+Between 2020-2023, membership in the R-Ladies global organization increased 
+while the availability of volunteers remained limited. At the conclusion of this
+time period several global team members retired. We are indebted for their 
+efforts that helped the organization thrive and grateful for their time spent 
+on the global team. 
+
+As the shift in personnel occurred, new teams and team structures were identified
```
Comment (yabellini) [inline content/news/2024-03-13-welcome-global-team/index.en.md:60]: the word personnel made me think on people we pay for their services. I would change for some word that don't have that meaning.  I would use volunteers.

**Review by yabellini — CHANGES_REQUESTED** [review]
Suggestion to change the word personnel for volunteers.  Personnel made me think on people that we pay for their services.

Ther est looks good.

File: content/news/2024-03-13-welcome-global-team/index.en.md (hunk 9 above)
Comment (shannonpileggi) [reply content/news/2024-03-13-welcome-global-team/index.en.md]: great catch, thank you! updated!

**Review by yabellini — APPROVED** [review]
Thank you for writing this :-)

**Pull request merged**